topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,718 @@
+
+> ### {% icon tip %} Tip: Learn more on chromosome conformation
+> To learn more about chromosome conformation and TADs, you can follow our [HiC tutorial]({{site.baseurl}}/topics/epigenetics/tutorials/hicexplorer/tutorial.html)
+{: .tip}
+
+In mammals, the X chromosome inactivation (XCI) balances the dosage of X-linked genes between females and males. The genes on the inactive X (Xi) chromosome are not expressed. 
+
+Some changes in the histone modifications could be involved. For example, the H3K4m3 is adding 3 methyl-group of the 4th Lysine in the histone 3 amino-acid. This modification is known to activate the transcription on nearby genes by opening the chromatine. The H3K27me3 on the other hand is inactivating the transcription of the nearby genes:
+
+![Fadloun et al, 2013](../../images/formation_of_super-structures_on_xi/histone_modifications.jpg "Source: Fadloun et al, 2013")
+
+More H3K27me3 and less H3K4m3 on the Xi could explain a lower expression of the genes there. 
+
+It has been also observed that the Xi adopts a distinct conformation without evident compartments or TAD. Cohesins, condensins and CCCTC-binding factor (CTCF) play key roles in chromosomal architectures and TAD formation which are other potential cause of the repression of the expression of the genes on Xi.
+
+The structural-maintenance-of-chromosomes hinge domain containing 1 (SMCHD1) has been found enriched on the Xi. It may be the potential actor in the shape of Xi and the change in gene expression there.
+
+[Wang et al. (2018)](https://www.cell.com/cell/fulltext/S0092-8674(18)30584-1) investigates the mechanism by which the SMCHD1 gene shapes the Xi and represses the expression of the genes on Xi in mouse.
+
+Their idea was to identify the differences which could be observed between the Xi and activated X chromosome, on both wild-type and SMCHD1 gene knockdown samples to study the SMCHD1 effect.
+In different experiments, they targetted histones with H3K27me3 or H3K4me3 and CTCF using ChIP-seq experiments:
+
+![ChIP-seq workflow](../../images/formation_of_super-structures_on_xi/Chromatin_immunoprecipitation_sequencing.jpg "Source: http://e.biohackers.net/ChIP-seq")
+
+They obtained sequences corresponding to portion of DNA linked to histones with H3K27me3, H3K4me3 or CTCF are found. Using this information, they could identify if there are differences in the H3K27me3, H3K4me3 and CTCF between the X (active or inactive) chromosomes and the potentially influenced genes.
+
+In the upcoming tutorial, we will reproduce the analysis of the ChIP-seq data step by step:
+
+- CTCF with 2 replicates: `wt_CTCF_rep1` and `wt_CTCF_rep2`
+- H3K4me3 with 2 replicates: `wt_H3K4me3_rep1` and `wt_H3K4me3_rep2`
+- H3K27me3 with 2 replicates: `wt_H3K27me3_rep1` and `wt_H3K27me3_rep2`
+- 'input' with 2 replicates: `wt_input_rep1` and `wt_input_rep2`
+
+    In 'input' samples, the same treatment as the ChIP-seq samples was done except for the immunoprecipitation step.
```

Comment:
It would be nice to have at least in the end a comparison of active and inactive X chromosome ChIP-Seq data to see the difference between them.
Clarify that in the following only inactivate (at least I assume so) X is discussed.